src/main/java/unicash/model/category/Category.java

```diff
@@ -60,4 +62,14 @@ public int hashCode() {
     public String toString() {
         return category;
     }
+
+    /**
+     * Returns category with a prefixed hashtag
+     */
+    public String hashTagToString() {
```

Comment:
Small nit, the naming of this method might be unintuitive. I almost intepreted it as get the hash value of the tag and return the string

Reply:
you're right I'll get that fixed